src/addTodoForm.js

```diff
@@ -1,18 +1,24 @@
-import React from "react";
+import React, { useState } from "react";
 
-function AddTodoForm(props) {
+function AddTodoForm({onAddTodo}) {
+
+    const [todoTitle, setTodoTitle] = useState('')
+
+    function handleTitleChange(event) {
+        const newTodoTitle = event.target.value
+        setTodoTitle(newTodoTitle)
+    }
 
     function handleAddTodo(event) {
         event.preventDefault()
-        const todoTitle = event.target.title.value
-        props.onAddTodo(todoTitle)
-        event.target.reset()
+        onAddTodo({ title: todoTitle, id: Date.now() })
+        event.target.title.value = ""
     }
 
     return (
         <form onSubmit={handleAddTodo}>
             <label htmlFor="todoTitle">Todo:</label>
-            <input name="title" id="todoTitle" />
+            <input name="title" id="todoTitle" onChange={handleTitleChange} />
```

Comment:
Requires changes:
Since this input is now updating React state via `onChange={handleTitleChange}`, it should also be assigned a `value` property of `todoTitle` to make the input controlled. 
> - [ ]  Add value prop equal to todoTitle from component props

Without this value prop, the `input` is only partially controlled and you'll get an error in the console:
<img width="1721" alt="Screenshot 2023-10-19 at 9 17 19 AM" src="https://github.com/Gio-Romero/react-todo/assets/22607722/c7d4c0ef-8f7c-49b4-bf5e-15135babfc1b">